css/gallery.css

```diff
@@ -0,0 +1,116 @@
+/*class for the section of gallery*/
+.info-gallery {
+    width: 635px;
+    height: 88px;
+    font-family: Roboto;
+    font-size: 16px;
+    font-weight: normal;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: normal;
+    letter-spacing: normal;
+    color: black;   
+}
+.info-gallery__text {
+    top: 0;
+    left: 0;
+    margin-right: 150px;
+    margin-left: 150px;
+    margin-bottom: 17px;
+}
+ .container-images{
+    display: flex;
+    flex-flow: row wrap;
+ }
+ .container-images :nth-child(4){
+    margin-right: 0px;  
+ }
+ .container-images :nth-child(8){
```

Comment:
What are you doing in these lines?

Reply:
He removed the right margin to the container of images 4 and 8

Reply:
its correct, you can make it a little shorter by replacing this 2 blocks with:

```
.container-images :nth-child(4),
.container-images :nth-child(8) {
    margin-right: 0px;  
 }
```

also please be sure to ad a space between al selectors and the bracket.